plugins/rollbar_dashboards/lib/rollbar_dashboards/client.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+module RollbarDashboards
+  class Client
+    MAX_RQL_JOB_WAIT_TIME = 5
+
+    def initialize(dashboard)
+      @base = dashboard.base_url
+      @token = Samson::Secrets::KeyResolver.new(dashboard.project, []).resolved_attribute(dashboard.read_token)
+    end
+
+    # returns array hashes of top items
+    def top_errors(hours: 24, environments: ['production'])
+      args = {
+        access_token: @token,
+        hours: hours,
+        environments: environments.join(',')
+      }.to_query
+
+      response = Faraday.get("#{@base}/reports/top_active_items?#{args}") do |req|
+        req.options.timeout = req.options.open_timeout = 1
+      end
+
+      if response.success?
+        data = ::JSON.parse(response.body, symbolize_names: true)
+        data[:result].map { |item_hash| item_hash[:item] }
```

Comment:
FYI usually nice to avoid nils via `data.fetch(:result).map { |i| i.fetch(:item) }`